README.md

```diff
@@ -89,9 +85,57 @@ affect_receive:
 
 This config injects a fault to node B at time 20 to delay its receipt of messages on `topic1`. Please see this [fault_injection.py](graph_generator/fault_injection.py) for a full list of faults supported.
 
-## Example Configs And Dataset
+## Example Configs
+
+In the [config](graph_generator/config/autonomous_vehicle/) folder, we have created an example graph that approximates autonomous vehicle software architecture. You can visualize how faults propagate with the following command:
+
+```bash
+bazel run //graph_generator:main -- --graph graph_generator/config/autonomous_vehicle/graph.yaml --node_feature_output ~/out --fault graph_generator/config/autonomous_vehicle/faults/crash_camera_driver1.yaml --fault_label_output /tmp/a --stop 200 --inject_at 10 --viz
+```
+
+> ⚠️ **Warning**:: Using `--viz` can slow down execution significantly therefore when this flag is present, dataset generation is turned of.
+
+## Generating Datasets
+
+The executor is capable of generating a large number of datasets based on configurations. Users can sweep over define faults and inject them at various time. For example, by running the following command:
+
+```bash
+bazel run //graph_generator/dataset:generate_datasets -- --graph graph_generator/config/autonomous_vehicle/graph.yaml --output_dir ~/output  --fault_dir graph_generator/config/autonomous_vehicle/faults --stop 1500 --fault_begin 800 --fault_end 1100 --max_num_sweep 30
+```
+
+the executor injects all faults specified in the fault_dir directory into the graph (defined by graph.yaml), and sweeps the injection time between 800 adn 1100 time units with equal partitions (800,810,820...). Sweeping the fault injection time may allow models to learn fault transition better. The output files have the following structure:
```

Comment:
nit: "between 800 **and** 1100 time units"

Reply:
```suggestion
the executor injects all faults specified in the fault_dir directory into the graph (defined by graph.yaml), and sweeps the injection time between 800 and 1100 time units with equal partitions (800,810,820...). Sweeping the fault injection time may allow models to learn fault transition better. The output files have the following structure:
```